src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/AdjudicationMigrateDto.kt

```diff
@@ -86,6 +89,17 @@ data class MigrateWitness(
   val witnessType: WitnessCode,
 )
 
+@Schema(description = "damages recorded on adjudication - repair in nomis")
+data class MigrateDamage(
+  @Schema(description = "damage / repair type")
+  val damageType: DamageCode,
```

Comment:
seems nomis should map to our codes based on dev

our codes:

ELECTRICAL_REPAIR,
  PLUMBING_REPAIR,
  FURNITURE_OR_FABRIC_REPAIR,
  LOCK_REPAIR,
  REDECORATION,
  CLEANING,
  REPLACE_AN_ITEM,

Nomis ref data

CLEA	Cleaning
DECO	Re-Decoration
ELEC	Electrical
FABR	Fabric
LOCK	Lock
PLUM	Plumbing